client/commands/command.py

```diff
@@ -69,11 +69,11 @@ def __init__(
         self._original_directory = arguments.original_directory
         self._current_directory = arguments.current_directory
 
-    def _run(self) -> None:
+    def _run(self) -> int:
         pass
 
-    def run(self) -> None:
-        self._run()
+    def run(self) -> int:
+        return self._run()
```

Comment:
I personally like returning a status a little more. I think it's better to be explicit that _run() can return status and to show what that status is in each command.

I'm totally fine with the method you listed, though, if you or anyone else thinks that's the better way.

Reply:
I think that an explicit return type makes the intent much more evident. When debugging or understanding the code, it's much easier to track `return` statements rather than calls to other functions.